src/hooks/v2v3/transactor/SetNftOperatorPermissionsTx.tsx

```diff
@@ -0,0 +1,53 @@
+import { getAddress } from '@ethersproject/address'
+import { t } from '@lingui/macro'
+import { ProjectMetadataContext } from 'contexts/projectMetadataContext'
+import { TransactionContext } from 'contexts/transactionContext'
+import { V2V3ContractsContext } from 'contexts/v2v3/V2V3ContractsContext'
+import {
+  handleTransactionException,
+  TransactorInstance,
+} from 'hooks/Transactor'
+import { useWallet } from 'hooks/Wallet'
+import { V2OperatorPermission } from 'models/v2v3/permissions'
+import { useContext } from 'react'
+import invariant from 'tiny-invariant'
+
+export function useSetNftOperatorPermissionsTx(): TransactorInstance {
```

Comment:
**suggestion** can we at least make this a generic `useSetOperator` that accepts the relevent args? Then pass the nft args in?

Reply:
actually don't worry about this. Non-blocking, can fix later.